### PR TITLE
Update how parameters, inital people in each compartment and contact …

### LINKED ISSUE #78: Correct python examples
Update python examples to new syntax how parameters, contact patterns and initial compartment populations are defined.

### PR DESCRIPTION
**Pull request by lelange**
…patterns are defined. Open questions are marked with TODO.

Closes #78 

## Merge Request - GuideLine Checklist 

**Guideline** to check code before resolve WIP and approval, respectively.
As many checkboxes as possible should be ticked.

### Checks by code author:
* [x] There is at least one issue associated with the pull request.
* [x] The branch follows the naming conventions as defined in the [git workflow](git-workflow).
* [ ] New code adheres with the [coding guidelines](coding-guidelines)
* [ ] Tests for new functionality has been added
* [x] A local test was successful
* [ ] There is appropriate **documentation** of your work. (use doxygen style comments)
* [ ] If new third party software is used, did you pay attention to its license? Please remember to add it to the wiki after successful merging.
* [ ] If new mathematical methods or epidemiological terms are used, has the glossary been updated ? Did you provide further documentation ?
 is present or referenced. Please provide your references.
* [ ] The following questions are addressed in the documentation*:  Developers (what did you do?, how can it be maintained?), For users (how to use your work?), For admins (how to install and configure your work?)
* For documentation: Please write or update the Readme in the current working directory!

### Checks by code reviewer(s):
* [ ] Is the code clean of development artifacts e.g., unnecessary comments, prints, ...
* [ ] The ticket goals for each associated issue are reached or problems are clearly addressed (i.e., a new issue was introduced).
* [ ] There are appropriate **unit tests** and they pass.
* [ ] The git history is clean and linearized for the merge request.
* [ ] Coverage report for new code is acceptable.

### CODE REVIEW

**Comment by mknaranja** [comment]
Where does "…patterns are defined. Open questions are marked with TODO." come from?

File: pycode/examples/secir_groups.py
Comment (mknaranja) [inline pycode/examples/secir_groups.py:84]: No, not necessarily. As all parameters, it has a default value. But it is better to explicitly call the setters for all our parameters in the example. Please remove comment.

File: pycode/examples/secir_groups.py
Comment (mknaranja) [inline pycode/examples/secir_groups.py:91]: Please just set it as a dummy to twice the RiskOfInfectionFromSymptomatic. It is only used if the test-and-trace-capacity is used. You can add this as a comment. Good examples on the parameters that can be used should be get from the cpp tests: https://github.com/lelange/memilio/blob/main/cpp/tests/test_secir.cpp and https://github.com/lelange/memilio/blob/main/cpp/tests/test_secir_ageres.cpp

File: pycode/examples/secir_groups.py
Comment (mknaranja) [inline pycode/examples/secir_groups.py:98]: Please stick to the same values in the examples (0.9 before, 0.7 now) if there isn't any reason to change.

File: pycode/examples/secir_simple.py
Comment (mknaranja) [inline pycode/examples/secir_simple.py:82]: see above

File: pycode/examples/secir_simple.py
Comment (mknaranja) [inline pycode/examples/secir_simple.py:84]: model.parameters.StartDay should do this.

File: pycode/examples/secir_simple.py
Comment (mknaranja) [inline pycode/examples/secir_simple.py:86]: I don't understand.

**Review by mknaranja — COMMENTED** [review]
@lelange I hope that my answers were clear. 
@dabele Could you check once the changes are made and approve/merge if this is done still this week?

File: pycode/examples/secir_groups.py
Comment (lelange) [reply pycode/examples/secir_groups.py:98]: I used the values that are in the testing example assuming it should be equal everywhere.

File: pycode/examples/secir_groups.py
Comment (dabele) [inline pycode/examples/secir_groups.py:93]: `(date(start_year, start_month, start_day) - date(start_year, 1, 1)).days` should do it. Same below.

**Review by dabele — CHANGES_REQUESTED** [review]

**Review by dabele — APPROVED** [review]

**Pull request merged**
